infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -706,7 +706,7 @@ private boolean clusterRoleChanged(TopologyDescriptor discoveredTopology) {
      * @param discoveredTopology new discovered topology
      */
     private void onStandbyClusterAddRemove(TopologyDescriptor discoveredTopology) {
-        log.debug("Standby Cluster has been added or removed from topology={}", discoveredTopology);
+        log.debug("Standby Cluster has been changed");
```

Comment:
I agree we don't have to log the topology info again as it is already printed in CorfuReplicationDiscoveryService.java line 664. However, I believe this message should be more informative and keep it as "added or removed". Otherwise, what does "changed" mean? that some configuration within an existing standby cluster changed?